bugguide/bugguide
# #156 Add respective indentation

### Description

Before:

![](https://user-images.githubusercontent.com/1582129/102513645-78ed0b80-4059-11eb-9e66-fbc40ca39830.png)

After:  

![image](https://user-images.githubusercontent.com/1582129/102513558-5c50d380-4059-11eb-9fbd-367a37673cae.png)

### Testing steps

*   [ ] Log in as admin
*   [ ] Go to /bg/orphans
*   [ ] You should see now that the report is well formated instead of a single paragraph report which is confusing (See above screenshots)

### Notes

This PR was created from branch #147 since I need those changes to be applied first that's why you see more commit changes.
